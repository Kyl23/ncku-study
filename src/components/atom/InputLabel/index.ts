import style from '@emotion/styled';
import InputLabel from '@mui/material/InputLabel';
import { color } from '~/src/theme/global';

export default style(InputLabel)`
    &.MuiInputLabel-shrink {
        font-size: 2.2rem;
    }

    &.Mui-focused {
        color: ${color.darkYellow};
    }
`;
