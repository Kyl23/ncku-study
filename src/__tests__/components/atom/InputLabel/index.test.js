import { render } from '@testing-library/react';
import InputLabel from '~/src/components/atom/InputLabel';

describe('test for atom - InputLabel', () => {
    it('render InputLabel without props must be same', () => {
        const { container } = render(<InputLabel />);

        expect(container).toMatchSnapshot();
    });
});
