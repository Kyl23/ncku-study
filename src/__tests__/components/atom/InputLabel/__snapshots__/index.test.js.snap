// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test for atom - InputLabel render InputLabel without props must be same 1`] = `
<div>
  <label
    class="MuiInputLabel-root MuiInputLabel-animated MuiFormLabel-root MuiFormLabel-colorPrimary css-1hw88bb-MuiFormLabel-root-MuiInputLabel-root"
  />
</div>
`;
